utils.test.js

```diff
@@ -31,5 +31,8 @@ describe('utils', function() {
   it('checkUrl', () => {
     expect(checkUrl(['wiki.jenkins.io'], 'https://wiki.jenkins.io/something')).toBe(true);
   });
+  it('removeExternalLink', () => {
```

Comment:
Added a basic test, is there a way I can run and check if this works?

Reply:
`npm run test` should do it.

Reply:
Thanks